### PR TITLE
full index manual: add missing pages

### PR DESCRIPTION
**Pull request by neteler**
- add "wxGUI" section to list wxGUI related manual pages
- add "Further pages" section to list "variables" manual page
- minor http to https URL fixes

Fixes https://trac.osgeo.org/grass/ticket/3987

### CODE REVIEW

**Comment by veroandreo** [comment]
What about "Other" instead of "Further"?

File: man/build_html.py [hunk 1]
```diff
@@ -241,8 +255,8 @@
 <li class="toc"><a class="toc" href="full_index.html#r3">3D raster commands (r3.*)</a></li>
 <li class="toc"><a class="toc" href="full_index.html#t">Temporal commands (t.*)</a></li>
 <li class="toc"><a class="toc" href="full_index.html#v">Vector commands (v.*)</a></li>
-<li class="toc"><a class="toc" href="wxGUI.html">wxGUI Graphical User Interface</a></li>
-<li class="toc"><a class="toc" href="wxGUI.nviz.html">3D visualization suite</a></li>
+<li class="toc"><a class="toc" href="full_index.html#wxGUI">wxGUI Graphical User Interface</a></li>
+<li class="toc"><a class="toc" href="full_index.html#further">Further pages</a></li>
```
Comment (neteler) [inline man/build_html.py]:
```suggestion
<li class="toc"><a class="toc" href="full_index.html#other">Other pages</a></li>
```

File: man/build_html.py [hunk 2]
```diff
@@ -194,7 +194,21 @@
...
 footer_tmpl = string.Template(\
-r"""<hr class="header">
+r"""<a name="wxGUI"></a>
+<h3>wxGUI: Graphical user interface</h3>
+<table><tbody>
+<tr><td valign="top"><a href="wxGUI.html">wxGUI Graphical User Interface</a></td> <td>wxGUI Graphical User Interface</td></tr>
+<tr><td valign="top"><a href="wxGUI.nviz.html">3D visualization suite</a></td>    <td>wxGUI.nviz 3D visualization suite</td></tr>
+</tbody></table>
+<p>
+<a name="further"></a>
```
Comment (neteler) [inline man/build_html.py]:
```suggestion
<a name="other"></a>
```

File: man/build_html.py [hunk 3]
```diff
@@ -194,7 +194,21 @@
...
+<h3>Further pages</h3>
```
Comment (neteler) [inline man/build_html.py]:
```suggestion
<h3>Other pages</h3>
```

**Comment by neteler** [comment]
BTW: more pages to list there?

**Pull request merged**